### ISSUE TITLE
Timed Out Bewerbung

### ISSUE THREAD

**Issue by GDellAquila** [neopostmodern/wohnungsbot#28]
Hi,

after the bot starts working, every 'Bewerbungen' is showed 'Timed out'.

How can I fix it?

G. Dell'Aquila

**Comment by neopostmodern** [comment]
are you using the 1.2.0? which OS?

where does it get "stuck"? i.e., what is visible right before it times out? could you maybe attach a screenshot?

**Comment by neopostmodern** [comment]
Closing due to inactivity.

**Comment by jukoson** [comment]
I think I do have a similar issue. The Bot applies to an apartment and I receive an email, that it applied. However, looking into my immoscout postbox I do not see any messages sent by the bot. 
It has only applied to 2 apts so far, and the first one just timed out.
OS: Centos7
Version: 1.40
![image](https://user-images.githubusercontent.com/38342694/154269676-6abfb6d5-b0ca-4700-9754-8ae39af5972b.png)

**Comment by jukoson** [comment]
@neopostmodern For both applications I've received a message from the Bot, but not the regular IS24 email as in "Ihre Vorstellung beim Anbieter...".

For one of the applications I was able to see that the application was completed successfully, though I do not remember whether I had to type a Captcha.
I forgot to mention that I am running with --no-sandbox - so maybe I'll fix my environment first before keeping you busy further.
